### PR TITLE
Peter/expose geography hierarchy order

### PR DESCRIPTION
**Pull request by phynes-sensiblecode**

### CODE REVIEW

File: ctb_metadata_files/metadata.graphql [hunk 1]
```diff
@@ -33,6 +33,7 @@ type ONS_Variable {
   Variable_Type: Variable_Type!
   Quality_Statement_Text: String
   Quality_Summary_URL: String
+  Geography_Hierarchy_Order: String
```
Comment (duncanharris) [inline ctb_metadata_files/metadata.graphql:36]: Should this be non null?

**Review by mikejamesthompson — APPROVED** [review]
I've tested locally and looks good.

File: ctb_metadata_files/metadata.graphql (hunk 1 above)
Comment (phynes-sensiblecode) [reply ctb_metadata_files/metadata.graphql:36]: No, it will be null for non-geographic variables.

Comment (duncanharris) [reply ctb_metadata_files/metadata.graphql:36]: Yes, realised that after a few minutes.

**Pull request merged**